frontend/src/fetchers/helpers.js

```diff
@@ -0,0 +1,13 @@
+import join from 'url-join';
+
+const getReportsDownloadURL = (reportIds) => {
```

Comment:
This function calculates the URL, but doesn't perform a fetch/xhr request because we need to trigger a download via the browser.